### ISSUE TITLE
Use `StartStopEventWrapper` for sensor events

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#19131]
### What would you like to be added

Currently most sensors use a field + event combo to ensure the sensor tracking is started when the first subscriber is added and stopped when the last subscriber is removed. This logic is encapsulated in `StartStopEventWrapper`, so to simplify the codebase, it should be used instead.

**Comment by nickodei** [comment]
Hey, I would be down to implement this change. 

Just to make sure. You mean the files under:  `Uno.Ui.netcoremobile -> Devices\Sensors` ?

**Comment by nickodei** [comment]
@MartinZikmund Already embarked on this but fine, didn't matter. 

Isn't it then implemented by your changes? What needs to be done?

**Comment by MartinZikmund** [comment]
No, my changes just improve the `StartStopEventWrapper` API slightly, not in major way - should be easy to bring your changes up to date after it gets merged
